posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,615 @@
+  </featureManager>
+----
+    
+To find out more, take a look at the MicroProfile 6.0 https://download.eclipse.org/microprofile/microprofile-6.0/microprofile-spec-6.0.html[specification], https://github.com/eclipse/microprofile/releases/tag/6.0[release], and link:{url-prefix}/docs/latest/reference/javadoc/microprofile-6.0-javadoc.html[Javadoc].
+
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24759
+// Contact/Reviewer: gjwatts,ReeceNana
+// // // // // // // // 
+[#java20]
+== Support for Java SE 20
+
+Java 20 includes the following features and changes:
+
+* 429: link:https://openjdk.org/jeps/429[Scoped Values (Incubator)]
+* 432: link:https://openjdk.org/jeps/432[Record Patterns (Second Preview)]
+* 433: link:https://openjdk.org/jeps/433[Pattern Matching for switch (Fourth Preview)]
+* 434: link:https://openjdk.org/jeps/434[Foreign Function & Memory API (Second Preview)]
+* 436: link:https://openjdk.org/jeps/436[Virtual Threads (Second Preview)]
+* 437: link:https://openjdk.org/jeps/437[Structured Concurrency (Second Incubator)]
+
+
+To use Java 20: 
+
+1. link:https://adoptium.net/temurin/releases/?version=20[Download Java 20].
+
+2. Obtain the <<run,23.0.0.3>> version of Open Liberty.
+
+3. Edit your Liberty link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html#server-env[server.env file] to point `JAVA_HOME` to your Java 20 installation.
+
+For more information on Java 20, reference the Java 20 link:https://jdk.java.net/20/release-notes[release notes page], link:link:https://docs.oracle.com/en/java/javase/20/docs/api/index.html[API Javadoc page], link:https://adoptium.net/temurin/releases/?version=20[download page] or link:https://docs.oracle.com/en/java/javase/20/migrate/toc.htm[Java 20 migration guide].
+
+
+NOTE: To try out Java 20 preview features in Open Liberty, make sure to compile with `--enable-preview` and add the same parameter to your link:{url-prefix}/docs/latest/reference/directory-locations-properties.html[`jvm.options` file].
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+[#bugs]
+== Notable bugs fixed in this release
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23003+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.3].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24651[Liberty Server Hang randomly]
++
+Liberty Server hang happened randomly within docker containers which are deployed in k8s cluster in an Integration Pipeline. The javacore dumps were collected to analyze the issue. 
+In one javacore, one deadlock was identified and all operational threads are blocked due to the deadlock
+```

```

Comment:
i think you need a plus (+)  here to bring example under bullet and enable the other plus formatting at line 512